tutorials/darcy_thermo_mech/step04_velocity_aux/problems/step4.i

```diff
@@ -14,15 +14,15 @@
 
 [AuxVariables]
   [./velocity_x]
-    order = CONSTANT
+    order = FIRST
```

Comment:
I don't like the use of higher order MONOMIALS here.  This is supposed to be a tutorial to show people best-practices and I fear this will lead to confusion / bad habits.

While this happens to work in this case (because the pressure solution is linear) it will NOT be exact for any other type of pressure field.  It will be a projection and then the solution will be slightly wrong.

We should continue to couple to the gradient like it originally was: and only use the Aux system to "see" a projection of the velocity for postprocessing / visualization purposes.